app/views/admin/programs/_form.html.haml

```diff
@@ -9,6 +9,7 @@
       = f.input :schedule_fluid, label: "Allow submitters to change their event after it is scheduled"
       = f.input :rating, hint: 'Enter the number of different rating levels you want to have for voting on proposals. Enter 0 if you do not want to vote on proposals.'
       = f.input :languages, hint: "Enter the languages allowed for events as values of #{link_to('ISO 639-1', 'http://www.loc.gov/standards/iso639-2/php/code_list.php', target: "_blank")} language codes separated with commas. The first language would be the default language. Leave it blank if you do not want to specify languages.".html_safe
+      = f.input :schedule_cell_size, hint: "It is minimal time slots of your program. The value should be greater than 5, less than 60 and multiple of 60."
```

Comment:
I think the hint is not very clear. I would rewrite it, what about this:

_It is the minimal time interval of your schedule. The value should be 5, 6, 10, 12, 15, 20, 30 or 60_